packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -1977,7 +1984,7 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
       _CreationLocation location = knownLocations[id]!;
       expect(location.file, equals(file));
       // ClockText widget.
-      expect(location.line, equals(53));
+      expect(location.line, equals(60));
```

Comment:
Why are these values changing?

Reply:
I agree it's odd, but I think the reason is that this test is fragile: it seems to look at the reloaded code, which includes the test code itself, so if I modify the test, it modifies the line numbers in this test.

So, adding seven lines at the top of this file altered these values by seven lines.

Reply:
What an odd test 😄. If that is the case then I guess this is the right solution to keeping it working properly.